### PR TITLE
Release 2.0

### PR DESCRIPTION
**Pull request by nvie**
This PR is the merge target for everything that will land in the upcoming v2.0.0.

### CODE REVIEW

File: docs/pages/get-started/lexical-react.mdx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        LiveblocksPlugin,
+        FloatingComposer,
+        ThreadPanel,
+      } from "@liveblocks/react-lexical";
+
+      export default Editor() {
+        const room = useRoom();
+
+        // Wrap your initial config with `liveblocksConfig`
+        const initialConfig = liveblocksConfig({
```
Comment (nimeshnayaju) [inline docs/pages/get-started/lexical-react.mdx:146]: Should we also highlight the fact that users are expected to wrap their lexical configuration within `liveblocksConfig`? This function ensures `ThreadMarkNode` and `MentionNode` are correctly added to the editor config.

File: docs/pages/api-reference/liveblocks-react-lexical.mdx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+## useEditorStatus
+
+Returns the current editor status.
+
+```ts
+import { useEditorStatus } from "@liveblocks/react-lexical";
+
+const status = useEditorStatus();
+```
+
```
Comment (nimeshnayaju) [inline docs/pages/api-reference/liveblocks-react-lexical.mdx:118]: `<FloatingComposer />` does not currently need to be placed inside `LiveblocksPlugin` component. I wonder if we should still encourage this just to be safe.

File: docs/pages/api-reference/liveblocks-react-lexical.mdx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+  <LiveblocksPlugin>
+    <FloatingComposer />
+  </LiveblocksPlugin>
+</LexicalComposer>
+```
+
+{/* TODO: Image */}
+
+Submitting a comment will attach an annotation thread at the current selection.
+Should be nested inside [`LiveblocksPlugin`][].
```
Comment (nimeshnayaju) [inline docs/pages/api-reference/liveblocks-react-lexical.mdx:142]: `<FloatingComposer />` does not currently need to be placed inside `LiveblocksPlugin` component. I wonder if we should still encourage this just to be safe.

**Review by marcbouchenoire — APPROVED** [review]
🔜

**Pull request merged**